webui.py

```diff
@@ -224,35 +274,100 @@ def Start_WebUI(stable_diffusion_bridge_data, interrogation_bridge_data):
                 choices=["caption", "nsfw", "interrogation"],
                 value=interrogation_bridge_data.forms,
             )
-        gr.Button(value="Update Bridge", variant="Primary").click(
-            Update_Bridge,
-            inputs=[
-                horde_url,
-                worker_name,
-                api_key,
-                priority_usernames,
-                max_power,
-                queue_size,
-                max_threads,
-                nsfw,
-                censor_nsfw,
-                blacklist,
-                censorlist,
-                allow_img2img,
-                allow_painting,
-                allow_unsafe_ip,
-                allow_post_processing,
-                require_upfront_kudos,
-                dynamic_models,
-                number_of_dynamic_models,
-                max_models_to_download,
-                models_to_load,
-                models_to_skip,
-                forms,
-            ],
-            outputs=gr.TextArea(label="System Messages"),
+        with gr.Row():
+            system = gr.TextArea(label="System Messages", lines=1, interactive=False)
+        with gr.Row():
+            gr.Button(value="Toggle Maintenance Mode", variant="primary").click(
+                update_worker_mode, inputs=[worker_name, worker_ID, maintenance_mode, api_key], outputs=system
+            )
+            gr.Button(value="Update Bridge", variant="Primary").click(
+                Update_Bridge,
+                inputs=[
```

Comment:
Might be more clear to switch to a dictionary at this point for clarity in the code